test/e2e/csv_e2e_test.go

```diff
@@ -4432,89 +4426,86 @@ var csvFailedChecker = buildCSVConditionChecker(operatorsv1alpha1.CSVPhaseFailed
 var csvAnyChecker = buildCSVConditionChecker(operatorsv1alpha1.CSVPhasePending, operatorsv1alpha1.CSVPhaseSucceeded, operatorsv1alpha1.CSVPhaseReplacing, operatorsv1alpha1.CSVPhaseDeleting, operatorsv1alpha1.CSVPhaseFailed)
 var csvCopiedChecker = buildCSVReasonChecker(operatorsv1alpha1.CSVReasonCopied)
 
-func fetchCSV(c versioned.Interface, name, namespace string, checker csvConditionChecker) (*operatorsv1alpha1.ClusterServiceVersion, error) {
-	var fetchedCSV *operatorsv1alpha1.ClusterServiceVersion
-	var err error
+func fetchCSV(c versioned.Interface, name, namespace string, checker csvConditionChecker) (fetched *operatorsv1alpha1.ClusterServiceVersion, err error) {
```

Comment:
> (fetched *operatorsv1alpha1.ClusterServiceVersion, err error)
I don't typically see go return types given instance names.  Is there a case where that is preferred?

Reply:
So, AFAICT (based on testing and documentation) `Eventually()` is weird because it terminates the test function via an assertion, so the code following it is not necessarily invoked (i.e. the `return`). Naming the arguments seems to allow the error to be passed back to the caller of this function to assert on as well.
Without the named arguments, I was seeing panics on the return values when a timeout occured.

Reply:
I suppose one could argue that either:
a. `Eventually()` should be used to call this function rather than having `Eventually()` inside the function (there are dozens if not hundreds of invocations)
b. `wait.Poll()` should be used instead, taking advantage of the (existing) error handling by the caller.

Reply:
Named returns are one of the most insidious and pernicious features in Go, often seen as an anti-pattern. Perhaps we just use control flow that is grok-able and doesn't require this?

Reply:
Would you prefer `time.poll` then? `Eventually()` has some odd behaviors, admittedly.